numba/cuda/stubs.py

```diff
@@ -243,6 +243,17 @@ def local_array(shape, dtype):
     return ir.Intrinsic(fname, sig, args=(shape, dtype))
 
 
+class popc(Stub):
```

Comment:
Perhaps this should be added somewhere below in a new section as this section is marked `local_array` https://github.com/njwhite/numba/blob/dbb8471299795681cf124e0105ba00215a348169/numba/cuda/stubs.py#L234 ?

Reply:
Done!